src/sbt-test/sbt-accessibility-linter/e2e-with-defaults/src/test/scala/ExampleSpec.scala

```diff
@@ -33,19 +33,5 @@ class ExampleSpec extends flatspec.AnyFlatSpec with Matchers with AccessibilityM
       </html>
     """ should not (passAccessibilityChecks)
   }
-  
-  "Page with input with duplicate attribute" should "fail vnu checks" in {
-    s"""
-      <!DOCTYPE html>
-      <html lang="en">
-        <head><title>Example</title></head>
-        <body>
-          <main>
-            <h1>Example</h1>
-            <label>Example <input type="text" type="text"></label>
-          </main>
-        </body>
-      </html>
-    """ should not (passAccessibilityChecks)
-  }
```

Comment:
removed since we don't have vnu anymore, other scripted tests probably still have broken assertions